### PR TITLE
feat(securityscan): add multiple registries

### PR DESCRIPTION
**Pull request by pbalogh-sa**
Signed-off-by: Peter Balogh <p.balogh.sa@gmail.com>

| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    |yes
| API breaks?     | no
| Deprecations?   | no
| Related tickets | related https://github.com/banzaicloud/pipeline/pull/3403
| License         | Apache 2.0

### What's in this PR?
Implement supporting multiple image registries in `securityscan` integrated service.

### Checklist

- [x] Code meets the [Developer Guide](https://github.com/banzaicloud/developer-guide)

### CODE REVIEW

**Review by bonifaido — APPROVED** [review]

File: internal/cli/command/cluster/integratedservice/services/securityscan/common.go [hunk 1]
```diff
@@ -59,8 +59,10 @@ func (s ServiceSpec) Validate() error {
 
 	validationErrors = errors.Combine(validationErrors, s.WebhookConfig.Validate())
 
-	if s.Registry != nil {
-		validationErrors = errors.Combine(validationErrors, s.Registry.Validate())
+	if s.Registries != nil {
```
Comment (pregnor) [inline internal/cli/command/cluster/integratedservice/services/securityscan/common.go:62]: Note: unnecessary nil check (for loop does it automatically).

**Review by pregnor — APPROVED** [review]

File: internal/cli/command/cluster/integratedservice/services/securityscan/common.go (hunk 1 above)
Comment (pbalogh-sa) [reply internal/cli/command/cluster/integratedservice/services/securityscan/common.go]: done

File: internal/cli/command/cluster/integratedservice/services/securityscan/common.go [hunk 2]
```diff
@@ -38,7 +38,7 @@ type ServiceSpec struct {
 	Policy           policySpec        `json:"policy" mapstructure:"policy"`
 	ReleaseWhiteList []releaseSpec     `json:"releaseWhiteList,omitempty" mapstructure:"releaseWhiteList"`
 	WebhookConfig    webHookConfigSpec `json:"webhookConfig" mapstructure:"webhookConfig"`
-	Registry         *registrySpec     `json:"registry,omitempty" mapstructure:"registry"`
+	Registries       []*registrySpec   `json:"registry,omitempty" mapstructure:"registries"`
```
Comment (sagikazarmark) [inline internal/cli/command/cluster/integratedservice/services/securityscan/common.go:41]: See my comment in the API PR.

**Comment by pbalogh-sa** [comment]
I've tested it again, using a single registry and a list of registries as well.

**Review by pregnor — APPROVED** [review]

**Pull request merged**